client/src/components/typedefs/typedefs.tsx

```diff
@@ -48,6 +66,34 @@ export interface InitialValues {
   includeAdult: boolean,
 }
 
+export interface Action {
+  type: string;
+  locale?: string;
+}
+
+export interface AppContextType {
+  state: {
+    locale: string;
+  };
+  dispatch: Dispatch<Action>;
+}
+
+export interface AppContextProviderProps {
+  children: ReactNode;
+}
+
+export interface Errors {
+  listName?: string;
+}
+
+export interface Values {
+  listName?: string;
+}
+
+export interface MoviesByIds {
```

Comment:
Here just movies, mb would be better give a name like `Movies` ?

Reply:
I think no, because I already have one interface for just ```Movies```